regression-test/suites/json_p0/test_json_read_by_line.groovy

```diff
@@ -0,0 +1,177 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
since you are also testing json_root and json_path in this Groovy file, the file name is no longer sufficient to describe what you have done here.

Reply:
The point is to set enable_simdjson_reader=false and test  test  json_root, jsonpath and read_by_line, adding  enable_simdjson_reader=false to test_stream_load_properties.groovy may change the intention.

Reply:
@HowardQin Maybe change name of this file is better, for testing json_root ,read_json_by_line  and json_path.

Reply:
It will be moved to load_p0/stream_load/test_stream_load_properties.groovy